pkg/crc/machine/machine.go

```diff
@@ -67,6 +67,12 @@ func getCrcBundleInfo(bundlePath string) (*bundle.CrcBundleInfo, error) {
 	bundleName := filepath.Base(bundlePath)
 	bundleInfo, err := bundle.GetCachedBundleInfo(bundleName)
 	if err == nil {
+		if fmt.Sprintf(".%s", bundleInfo.ClusterInfo.AppsDomain) != constants.AppsDomain {
+			return nil, fmt.Errorf("unexpected bundle, it must have %s apps domain", constants.AppsDomain)
+		}
+		if bundleInfo.GetAPIHostname() != fmt.Sprintf("api%s", constants.BaseDomain) {
+			return nil, fmt.Errorf("unexpected bundle, it must have %s base domain", constants.BaseDomain)
+		}
```

Comment:
Ah, this is just validating the whatever we are putting on the bundle should be same what we put in the constants. Now if we ever want to change the snc side then here also we do need to change :( wouldn't it better to have this info at one place where the bundle is created?

Reply:
An alternative would be to browse all installed bundles to get all base domains and apps domains, but this is code doesn't exist yet! WDYT?

Reply:
I was thinking we'd add some markers in `/etc/hosts` in order to do the cleanup?
```
### These entries were automatically by CodeReady Containers (crc)
....
### End of entries automatically added by crc
```
We'll need to hardcode `*.crc.testing` during the transition.

Reply:
If I remove these 2 checks but keep for now hardcoded the domain for the clean, would you accept it for the next release ? 
For sure, we need to add markers but I would prefer not to do a big bang.

Meanwhile, I try to improve the code in bundle package. It could help us for that.